suite-native/message-system/src/messageSystemMiddleware.ts

```diff
@@ -5,18 +5,28 @@ import {
     messageSystemActions,
     categorizeMessages,
     getValidMessages,
+    selectMessageSystemConfig,
 } from '@suite-common/message-system';
-import { selectAccountsSymbols } from '@suite-common/wallet-core';
+import { deviceActions, selectAccountsSymbols, selectDevice } from '@suite-common/wallet-core';
 
-const isAnyOfMessageSystemAffectingActions = isAnyOf(messageSystemActions.fetchSuccessUpdate);
+const isAnyOfMessageSystemAffectingActions = isAnyOf(
+    messageSystemActions.fetchSuccessUpdate,
+    deviceActions.selectDevice,
```

Comment:
It seems, that there are also other actions that maybe can trigger the check - deviceChange, disconnect, forget, auth

Reply:
cc @matejkriz

Reply:
- deviceChanged and auth are not relevant (no relevant change for available message system conditions can be updated)
- disconnect and forget would also trigger selectDevice if selected, but not relevant if not selected